### ISSUE TITLE
Introduce an ErrorMessage component

### ISSUE THREAD

**Issue by benoitf** [containers/podman-desktop#1365]
### Is your enhancement related to a problem? Please describe

We're using at different places some errors to report to the user using some `<div>`

### Describe the solution you'd like

An Error component that will apply the common 'error' color.

### Describe alternatives you've considered

any other proposal allowing to set only once the color.

**Comment by cdrage** [comment]
Should we put this in a modal / template / maybe have a doc explaining how to do effective error reporting.

**Comment by deboer-tim** [comment]
I think this issue was only for a reusable UI element for the places we're currently showing errors in-context. I'm glad to starting working on a doc to cover how errors should be reported/standards, but if you're thinking of a full doc or modal dialog, etc then I'd prefer we open another issue.

**Comment by deboer-tim** [comment]
Closing, fixed via PR #1568 and PR #1598.
